public/json/beamline.json

```diff
@@ -24,7 +36,9 @@
               "width": "90%",
               "height": "20px",
               "textAlign": "center",
-              "fontWeight": "bold",
+              "font": {
+                "style": "bold"
+              },
```

Comment:
Ideally prop-types would flag these things.

Reply:
We should make a note of this kind of thing as an example of what prop types does not help us solve, so that we could compare it to another object -> type package in the future